_layouts/post.html

```diff
@@ -6,7 +6,7 @@
 {% include section/header.html %}
 
 {% if page.options contains 'full-bleed-cover' %}
-<div class="cover-image" style="background-image: url({{ page.cover | relative_url }});"></div>
+<div class="cover-image" style="background-image: url({{ page.cover | relative_url }});" {% if page.cover_attribution %} title="{{ page.cover_attribution }}" {% endif %}></div>
```

Comment:
Not a blocker, but title should probably be reserved for accessibility purposes (descriptive alt text, basically). Attribution should probably be a little visible below the image (however this will require some slight design tweaks).

Additionally this should probably have a `role="presentation"` to hide it from screen readers anyways.